packages/runner-ct/src/errors/script-error.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { namedObserver } from '../lib/mobx'
+const ansiToHtml = require('ansi-to-html')
+
+const convert = new ansiToHtml({
+  fg: '#000',
+  bg: '#fff',
+  newline: false,
+  escapeXML: true,
+  stream: false,
+})
+
+export const ScriptError: React.FC<{ error: string }> = namedObserver('ScriptError', ({ error }) => {
+  if (!error) return null
+
+  const errorHTML = convert.toHtml(error)
+
+  return (
+    <pre
+      className='script-error'
+      dangerouslySetInnerHTML={{ __html: errorHTML }}
```

Comment:
I would generally recommend using some library to sanitize input, but I don't know that we care here.

Reply:
One use case is the user has some unsafe html in the code, which manifests as a error message. Not too sure we care about that use case, though - that is, the user wrote some unsafe html in their own code, which might then somehow be rendered.